### PR TITLE
KAS-4700 show move subcase button only if there is no parliamentflow

### PR DESCRIPTION
**Pull request by kobemertens**
https://kanselarij.atlassian.net/jira/software/c/projects/KAS/boards/1?selectedIssue=KAS-4700

### CODE REVIEW

File: app/components/subcases/subcase-header.hbs [hunk 1]
```diff
@@ -8,44 +8,48 @@
...
+                  data-test-subcase-header-action-delete-subcase
+                  @skin="link"
+                  @alert={{true}}
+                  {{on "click" (fn this.requestDeleteSubcase @subcase)}}
+                  role="menuitem"
+                >
+                  {{t "delete-subcase"}}
+                </AuButton>
+              {{/if}}
+              {{#if @parliamentFlow}}
```
Comment (sergiofenoll) [inline app/components/subcases/subcase-header.hbs]: Might be misunderstanding ticket, shouldn't this be `{{#if (not @parliamentFlow)}}` → only show "move subcase" action if (sub)case is not linked to a parliament flow?

Comment (kobemertens) [reply app/components/subcases/subcase-header.hbs]: Whoops that was an `unless` I forgot to put back

Comment (kobemertens) [reply app/components/subcases/subcase-header.hbs]: fixed in [d033020](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/2118/commits/d033020fcd42e882bc29df4b9fe8f989e2595cff)

File: app/components/subcases/subcase-header.js [hunk 2]
```diff
@@ -84,6 +84,10 @@ export default class SubcasesSubcaseHeaderComponent extends Component {
     this.subcaseToDelete = subcase;
   }
 
+  get hasActions() {
+    return this.canDelete || this.canPropose || this.args.parliamentFlow;
```
Comment (sergiofenoll) [inline app/components/subcases/subcase-header.js]: @SchrodingersCat00 here as well, right? Should be `!this.args.parliamentFlow` I think.

Comment (kobemertens) [reply app/components/subcases/subcase-header.js]: You're right, not sure how I missed that

**Review by ValenberghsSven — APPROVED** [review]

**Pull request merged**